src/ngraph/frontend/onnx_import/exceptions.hpp

```diff
@@ -52,3 +59,8 @@ namespace ngraph
     } // namespace  onnx_import
 
 } // namespace  ngraph
+
+#define ASSERT_NOT_SUPPORTED(node_, cond_)                                                         \
+    NGRAPH_ASSERT_STREAM(ngraph::onnx_import::error::NotSupported, cond_) << node << " "
+#define ASSERT_VALID_ARGUMENT(node_, cond_)                                                        \
+    NGRAPH_ASSERT_STREAM(ngraph::onnx_import::error::InvalidArgument, cond_) << node << " "
```

Comment:
Typo? I believe you meant **`node_`** instead of *`node`*? You may also wrap `node_` with braces for the sake of safety.